src/Catalyst.Core.Modules.Web3/Controllers/Handlers/EthGetTransactionReceiptHandler.cs

```diff
@@ -21,18 +21,50 @@
 
 #endregion
 
+using System;
+using System.Linq;
 using Catalyst.Abstractions.Kvm.Models;
 using Catalyst.Abstractions.Ledger;
+using Catalyst.Abstractions.Ledger.Models;
+using Nethermind.Core;
 using Nethermind.Core.Crypto;
 
 namespace Catalyst.Core.Modules.Web3.Controllers.Handlers
 {
-    [EthWeb3RequestHandler("eth", "getCode")]
+    [EthWeb3RequestHandler("eth", "getTransactionReceipt")]
     public class EthGetTransactionReceiptHandler : EthWeb3RequestHandler<Keccak, ReceiptForRpc>
     {
-        protected override ReceiptForRpc Handle(Keccak param1, IWeb3EthApi api)
+        protected override ReceiptForRpc Handle(Keccak txHash, IWeb3EthApi api)
         {
-            throw new System.NotImplementedException();
+            if (api == null) throw new ArgumentNullException(nameof(api));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)